server.js

```diff
@@ -54,27 +57,36 @@ app.post('/api/ticket', (req, res) => {
   let numberOfSeats = req.body.ticket.numberOfSeats;
   let sessionId = req.body.ticket.sessionId;
 
-  Session.findOne({ _id: req.body.ticket.sessionId }, (err, data) => {
-    if (err) {
-      return res.status(500).json({ message: 'Запрос не выполнен' });
-    }
+  !ObjectId.isValid(sessionId)
+    ? res.status(400).json({ message: 'Невалидный ObjectId' })
+    : Session.findOne({ _id: req.body.ticket.sessionId }, (err, data) => {
+        if (err) {
+          return res.status(500).json({ message: 'Ошибка сервера' });
+        }
+        if (!data) {
+          return res.status(400).json({ message: 'Сеанс не найден' });
+        }
 
-    if (data.emptySeats < numberOfSeats) {
-      res.status(400).json({ message: 'Места закончились' });
-    } else {
-      const ticket = new Ticket({
-        name,
-        numberOfSeats,
-        sessionId
-      });
-
-      let error = ticket.validateSync();
-      if (error) {
-        let errorName = Object.keys(error.errors)[0];
-        res.status(400).json({
-          message: error.errors[errorName].message
+        if (data.emptySeats < numberOfSeats) {
+          return res.status(400).json({ message: 'Места закончились' });
+        }
+        const ticket = new Ticket({
+          name,
+          numberOfSeats,
+          sessionId
         });
-      } else {
+
+        let error = ticket.validateSync();
+
+        if (error) {
+          let errorMessages = [];
+          for (const key in error.errors) {
+            errorMessages.push(error.errors[key].message);
+          }
+          return res.status(400).json({
+            errorMessages
```

Comment:
You have different format for error here - array instead of message prop. How are you gonna handle that on UI?